egs/tedlium/s5_r2/RESULTS

```diff
@@ -0,0 +1,213 @@
+
+# GMMs
```

Comment:
In future you should probably include the commands used to obtain these, to make people's life easier in getting the results.  It's OK if the order it would be printed differs from what you have.  But I'll merge this as-is.